Add SQLite support (#178)
apache/iceberg-python: mkdocs/docs/configuration.md

    uri: postgresql+psycopg2://username:password@localhost/mydatabase
```

In the case of SQLite:

<!-- prettier-ignore-start -->

!!! warning inline end "Development only"

>     SQLite is not built for concurrency, and should only be used for exploratory or development purposes.

Reviewer: Avoid passive.

```suggestion
    SQLite is not built for concurrency, you should use this catalog for exploratory or development purposes.
```
